### PR TITLE
Fixed #1807. tasking manager panel wait features added to UI

### LINKED ISSUE #1807: tasking manager fixes to be prod ready
companion ticket for https://github.com/ngageoint/hootenanny/issues/4217

### PR DESCRIPTION
**Pull request by bobi7**
ref #1807

### CODE REVIEW

File: modules/Hoot/managers/api.js [hunk 1]
```diff
@@ -1428,6 +1428,30 @@ export default class API {
...
+            .then( resp => {
+                return {
+                    data: resp.data,
+                    message: 'Changeset upload complete.',
+                    status: 200,
+                    type: 'success'
+                };
+            } )
+            .catch( err => {
+                const message = err.data.message || 'Changeset upload failed.',
```
Comment (brianhatchl) [inline modules/Hoot/managers/api.js]: We don't show a user message for this request so is it necessary to format the response as one?

File: modules/Hoot/managers/api.js [hunk 2]
```diff
@@ -1514,6 +1538,31 @@ export default class API {
...
+                    type: 'success'
+                };
+            } )
+            .catch( err => {
+                return {
+                    message: err.data,
+                    status: err.status,
+                    type: err.type
+                };
+            } );
```
Comment (brianhatchl) [inline modules/Hoot/managers/api.js:1548]: Do we need the message format if we don't alert the user?

Comment (bobi7) [reply modules/Hoot/managers/api.js:1548]: We do alert user of this one

**Review by brianhatchl — APPROVED** [review]
:turtle: :turtle:

**Pull request merged**